### DISCUSSION TITLE
Can Strapi be used to power a SAAS product?

### DISCUSSION THREAD

**Discussion by holtmeister** [strapi/strapi#6029] [Legacy Topics]
Hi there, non technical business owner here! I'm wondering if Strapi could be used to develop a business SAAS which provides a number of tools to help run individual businesses... think customer support, knowledge base, chat, perhaps blogs and even newsletters and subscriptions etc. From research it seems that it could be possible to power the backend, with the frontend running an Angular app. However I'm unsure if Strapi is suited to this kind of use case? If so, would it be one instance of Strapi powering all the content, or would each 'business' have its own installation?

**Comment by danielehrhardt** [comment]
Why not?

**Comment by robertpiosik** [comment]
Keep in mind that strapi by its conveniences covers only a segment of everyday backend development tasks.

**Comment by lauriejim** [comment]
Hello, with the right data structure (depending of your needs) you will be able to achieve that with Strapi.
